oojs.md

```diff
@@ -24,13 +34,19 @@ Create an `Album` class that inherits from `Playlist`. It should also...
 - Have an `artist` (String) property that is determined by some input
 
 ```js
-Your answer goes here
+class Album extends Playlist={
+
+  artist:function(artist){
```

Comment:
`artist` should not be a method. It is a property on the `Album` class, meaning you can initialize it in this class' constructor.